CHANGELOG.md

```diff
@@ -2,6 +2,73 @@
+   - Added preliminary support for `xarray` to the `instrument` object
+   - Support for `today`, `tomorrow`, and `yesterday` as datetime objects
+   - Added `model_utils`, featuring preliminary support for data-model comparison
+   - Added support for 1d median in seasonal averages
+   - Added routine to convert from kp to Ap
+   - Added `pyglow` integration support for python 3.x
+   - Improves input handling for datetime parameters that are more precise than just year, month, and day, where appropriate
+   - Added merging routines to allow combination of measured and forecasted Kp and F10.7 indexes into a single instrument object
+ - Code restructure
+   - Moved instrument templates and methods to subdirectories
+   - Moved utils into multiple subdirectories to aid with organization
+ - Instrument Updates
+   - NASA CDAWeb download now uses https protocol rather than FTP
+   - `_instrument.py` supports xarray
+   - Support for listing files from remote server
+   - COSMIC RO data unified into single instrument object
+   - Added support for DEMETER IAP
+   - Added support for DMSP IVM Level 1 data
+   - Added routines to update DMSP ephemeris and drifts
+   - Added warnings to instruments without download support
+   - Added support for ICON FUV and MIGHTI
+   - Added support for JRO ISR
+   - Added support for F10.7 and more Kp forecast products
+   - Added instrument templates for Madrigal, CDAWeb, and netcdf_pandas
+   - Added support for TIMED SABER
+   - Added support for UCAR TIEGCM
+   - Switched download methods for CDAWeb and COSMIC data to use `requests`
+   - Removed support for SuperDARN and SuperMAG downloads while server changes are sorted out
+ - Updates to travis configuration
+   - Tests run for python 2.7 and 3.7
+   - Added display port to test plots
+ - Updates to community docs
+   - Added Issue templates
+   - Added Pull Request Template
+   - Added note for PR to be made to develop, not master
+ - Style updates throughout
+   - Consistent documentation for docstrings and instruments
+   - Cleaned up commented code lines
+   - PEP8 scrub
+ - Documentation
+   - Added FAQ section
+   - Added "powered by pysat" logo
+   - Updated supported instruments
+ - Unit Test Updates
+   - Dropped instrument templates from coverage
+   - Added multiple output options for `pysat_testing` object to aid with constellation tests. Removed old constellation test objects.
+   - Added test data for space weather indices to speed up testing
+   - Cyclic data for test instruments now generated from single test method
+   - test objects for xarray added
+   - Added test for parsed delimited files
+   - Removed ftp downloads from travis tests, still will run locally
+ - Bug fixes
+   - `pandas.ix` notation replaced with `pandas.loc` and `pandas.iloc` throughout
+   - Fixed a bug that forced user into interactive mode in `ssnl.plot`
+   - Bug fixes and cleanup in demo codes
+   - Fix for orbit iteration when data is not loaded
```

Comment:
Orbit problem related to a day with less than one orbit of data. Fix now covers multiple days with less than one orbit.

Reply:
updated